client/apps/edit/components/content2/sections/video/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react'
+import Controls from './controls.jsx'
+import Paragraph from '../../../../../../components/rich_text2/components/paragraph.coffee'
+import components from '@artsy/reaction-force/dist/components/publishing/index'
+const Video = components.Video
+const IconRemove = components.Icon.Remove
+
+export default class SectionVideo extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      progress: null
+    }
+    this.onCaptionChange = this.onCaptionChange.bind(this)
+    this.onRemoveImage = this.onRemoveImage.bind(this)
+    this.onClickOff = this.onClickOff.bind(this)
+    this.onProgress = this.onProgress.bind(this)
```

Comment:
These lines could be removed if we define the function with the fat arrow instead:
```
onCaptionChange = (html) => {
    this.props.section.set("caption", html)
}
```